lib.php

```diff
@@ -67,8 +71,8 @@ protected function update_init_params(array &$params, context $context,
         $PAGE->requires->js('/lib/editor/tinymce/plugins/tiny_mce_wiris/baseURL.js', false);
 
         // Add button after emoticon button in advancedbuttons3.
-        $added = $this->add_button_after($params, 3, 'tiny_mce_wiris_formulaEditor', '', false);
-        $added = $this->add_button_after($params, 3, 'tiny_mce_wiris_formulaEditorChemistry', '', false);
+        if ($editorisactive) $added = $this->add_button_after($params, 3, 'tiny_mce_wiris_formulaEditor', '', false);
```

Comment:
Rewrite this structure to follow best practices.
TravisCI tests should pass before submitting a pull request.